pkg/apic/provisioning/propertybuilder.go

```diff
@@ -44,6 +44,7 @@ type propertyDefinition struct {
 	IsEncrypted        bool                          `json:"x-axway-encrypted,omitempty"`
 	Widget             string                        `json:"x-axway-widget,omitempty"`
 	IsCopyable         bool                          `json:"x-axway-copyable,omitempty"`
+	ScopesUniqueItems  bool                          `json:"uniqueItems,omitempty"`
```

Comment:
the name of the attribute is misleading, as this attribute should be available for any ArrayProperty.

perhaps a new builder method needs added on that struct?